### PR TITLE
Metadata

### LINKED ISSUE #95: Create the `csv-metadata.json` files that defines the context

### PR DESCRIPTION
**Pull request by michael-sevcik**
Resolves #95

### CODE REVIEW

File: 4/csv/data.csv-metadata.json [hunk 1]
```diff
@@ -2,128 +2,503 @@
...
+                        "titles": "streetNameCS"
+                    },
+                    {
+                        "@type": "Column",
+                        "datatype": "string",
+                        "dc:description": "Name of a street in slovak",
+                        "name": "streetName",
+                        "propertyUrl": "ex:townName",
+                        "required": false,
+                        "titles": "streetNameSK"
```
Comment (michael-sevcik) [inline 4/csv/data.csv-metadata.json:63]: missing `"lang": "sk"`

**Review by michael-sevcik — COMMENTED** [review]
I really like the template

**Comment by patinoboh** [comment]
# Ok guys we need help.
The command `rdf serialize --input-format tabular --output-format turtle --minimal --decode-uri data.csv-metadata.json > output.ttl` finally **generates turtle file**. [The file](https://github.com/michael-sevcik/NPRG036-Data-Formats/blob/95-csv-metadata-json/4/csv/output.ttl) looks almost right, however it generates a **few weird lines** such as :
`<file:/home/patrik/Documents/datove_formaty/NPRG036-Data-Formats/4/csv/data-2.csv#occupantId> "Martin";`
and a few others at the end.
I think this problem occures from columns transformations where we do not use all columns for the transformation (mainly in _TownHallMap_ table and _Occupants_ table).
I have no idea how to fix that :( 
please help @michael-sevcik @ori-no @rastoboh

**Comment by patinoboh** [comment]
I think it is done now!

**Pull request merged**